k8s_helm_charts/zdm/values.yaml

```diff
@@ -0,0 +1,46 @@
+# Default values for zdm.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+namespace: zdmproxy
+
+proxy:
+  count: 3
+  logLevel: "INFO"
+  # two options: PRIMARY_ONLY and ASYNC_READS_ON_SECONDARY
+  readMode: "PRIMARY_ONLY"
+  # two options: ORIGIN and TARGET
+  primaryCluster: "ORIGIN"
+  maxClientConnections: "1000"
+  metricsEnabled: "true"
+  resources:
+    limits:
+      cpu: 4000m
+      memory: 8192Mi
+    requests:
+      cpu: 4000m
+      memory: 8192Mi
```

Comment:
8GB ram seems a bit too agressive for the proxy instances. We haven't benchmarked the memory usage of the proxy in a long time but the last time we did we didn't see usage go over 500MB. 1GB should be fine but we could also go for 2GB since we don't have recent data so it's better to be extra safe.

Reply:
changed to 2GB